DLPJTS-96 Update the README to specify a Maven command

datalogics-robl opened the pull request:
#### Changes in this pull request

- The README now gives better instructions on what Maven command to use.

#### Fulfills  [DLPJTS-96](https://jira.datalogics.com/browse/DLPJTS-96)

#### Checklist for approving this pull request

(PR creator amend this with more conditions if necessary)

- [x] There are **unit tests** for new/changed code, or a good explanation why this was not possible.
- [x] All **CI builders** have indicated success (Give them a few minutes to notice the pull request.)
- [x] The **Pull request title** has the JIRA issue numbers separated by spaces (if any), a space, and then a short, but descriptive summary.
- [x] **Commit messages** are well formed: [A note about Git commit messages](http://www.tpope.net/node/106)
- [x] New public packages, classes, and methods are **documented**. (Strongly consider documenting private classes and methods.)

#### Blockers

Add a checkbox for yourself with your **&#64;name** to this list if you are holding this PR open for review. PR Shepherd, please hold off merging this PR until these are all checked:

- [x] _&lt;add your name here with an **@**>_
- [x] @datalogics-bhaugen

datalogics-robl committed:
Update the README to specify a Maven command

We now tell the user to run './mvnw verify' to build the samples.

datalogics-devauto commented:
Build succeeded.

datalogics-devauto commented:
Build succeeded.

datalogics-bhaugen commented on README.md:
```diff
@@ -28,7 +28,7 @@ If this project was received as part of an evaluation, then the ``.use-pdfjt-lm`
 ## Building from the command line
 
-This project includes [maven-wrapper](https://github.com/datalogics/maven-wrapper), which fully encapsulates the build system required for PDF Java Toolkit. To build with Maven, simply type ``./mvnw`` at the command line (``.\mvnw`` on Windows).
+This project includes [maven-wrapper](https://github.com/datalogics/maven-wrapper), which fully encapsulates the build system required for PDF Java Toolkit. To build with Maven, simply type ``./mvnw verify`` at the command line (``.\mvnw verify`` on Windows).
```
@datalogics-robl, could we add something here to briefly explain what the ```verify``` goal for Maven does? How can the user determine that it was successful?

datalogics-robl replied:
I would love to add more clarity to this explanation, but I feel like we ought to discuss what the user is going to see once they've done everything in the README. What should the folder look like? How would they fire up a sample to do something? I'm a developer on this thing, and I have to confess its a little confusing right now even to me.

Rather than getting into a cycle of me tweaking the text, pushing the change, and waiting for your comments, let's sit down and decide what we'd like it to say.

I'll also point out that our documentation writer has already contributed changes to this file, and his input may be useful.

datalogics-kam replied:
I think it'd be good to link to [Maven doc](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html) on what the phases do.

Also, Maven makes a big report at the end that tells what succeeded and what didn't.

datalogics-robl committed:
More explanations in the README

The README now describes what the user will see after running mvnw, and
how to run the 'HelloWorld' sample from the constructed JAR.

datalogics-devauto commented:
Build succeeded.

datalogics-devauto commented:
Build succeeded.

datalogics-bhaugen commented:
@datalogics-robl 👍

datalogics-kam commented:
Thanks! 🐟 🍨
